internal/query/instance.go

```diff
@@ -453,14 +458,6 @@ func (i *authzInstance) ConsoleApplicationID() string {
 	return i.consoleAppID
 }
 
```

Comment:
Also cleanup the unused `host` and `domain` fields from the `authzInstance` struct.